start.py

```diff
@@ -0,0 +1,24 @@
+import os
+from os import path
+from shutil import copyfile
+
+def main(folder_name):
+   # Check if project already exist:
+   if path.exists(folder_name) == False :
+      # create folders/projects
+      cmd="scrapy startproject " + folder_name 
+      os.system(cmd)
+      current_dir = os.getcwd()
+      # coppy spider scripts from Spider_scripts folder
```

Comment:
surely we don't want to copy. What's making you do it?

Reply:
In this case I specifically wanted to copy files. I still don't have very much experience with scrapy, so may be missing something, but the issue is that **when you start a new project in scrapy, it automatically creates many folders and codes. In these automatically created folders, our spyders need to be in specific folder/place.**

So in short, I think I end up with two options:
- Leave automatically created folders/files in the git and put spyders where they need to be at the beginning
- Leave only my python spyders in git and provide a start script that will create scrapy projects and move my python spyders in the location where they need to be.

I opted for second option to leave git a bit cleaner.